Missing files

```
pollex/pollex-flask$ make start
make: *** No rule to make target 'instance/*', needed by 'pollex:dev'.  Stop.
```
Looks like `instance/*` is missing

--- a/pollex-flask/Makefile
+++ b/pollex-flask/Makefile
@@ -46,11 +46,17 @@ MAKE = make
 
 # Sources
 
-sources = run.py app/* config/* instance/*
+instance = instance/config.py
+
+sources = run.py app/* config/* $(instance)
 
 # Do it all
 
 all : $(RP_IMAGE)
+
+instance/config.py:
+	install -d instance
+	touch instance/config.py
 
 # Build RP image
 

